source/js/menu-burger.js

```diff
@@ -0,0 +1,7 @@
+let button = document.querySelector('.main-nav__toggle');
+let nav = document.querySelector('.main-nav');
```

Comment:
если переменная не будет переопределяться, всегда используй const
```suggestion
const button = document.querySelector('.main-nav__toggle');
const nav = document.querySelector('.main-nav');
```